src/test/mina_automation/mina_automation.ml

```diff
@@ -0,0 +1,12 @@
+(* mina_automation.ml -- test automation modules *)
+
+module Archive_blocks = Archive_blocks
```

Comment:
I think this file can be removed
Such a module will be auto-generated by ocaml compiler